.github/workflows/main.yml

```
@@ -41,3 +43,11 @@ jobs:
           name: conan-vs-extension.vsix
           path: bin\Release\conan-vs-extension.vsix
           if-no-files-found: error
+
+      - name: Publish release to marketplace
+        if: ${{ startsWith(github.ref, 'refs/tags/') }}
```

probably we would like this to run only for some specific pattern of the tag. Maybe `refs/tags/v2` or `refs/tags/2`?
```suggestion
        if: ${{ startsWith(github.ref, 'refs/tags/') }}
```